apps/payments/next/styles/global.css

```diff
@@ -0,0 +1 @@
+@import '../../../../libs/shared/styles/src/lib/index.css';
```

Comment:
This should also include the styles from `fxa-payments-server/src/styles/tailwind.css`.